buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/RandomizedTestingPlugin.groovy

```diff
@@ -57,35 +41,16 @@ class RandomizedTestingPlugin implements Plugin<Project> {
         }
     }
 
-    static void replaceTestTask(TaskContainer tasks) {
-        // Gradle 4.8 introduced lazy tasks, thus we deal both with the `test` task as well as it's provider
-        // https://github.com/gradle/gradle/issues/5730#issuecomment-398822153
-        // since we can't be sure if the task was ever realized, we remove both the provider and the task
-        TaskProvider<Test> oldTestProvider
-        try {
-            oldTestProvider = tasks.named('test')
-        } catch (UnknownTaskException unused) {
-            // no test task, ok, user will use testing task on their own
-            return
+    static void createUnitTestTask(TaskContainer tasks) {
+        // only create a unitTest task if the `test` task exists as some project don't make use of it.
+        tasks.matching { it.name == "test" }.all {
+            // We don't want to run any tests with the Gradle test runner since we add our own randomized runner
+            it.enabled = false
+            RandomizedTestingTask unitTest = tasks.create('unitTest', RandomizedTestingTask)
+            unitTest.description = 'Runs unit tests with the randomized testing framework'
+            unitTest.include("**/*Tests.class")
```

Comment:
I think which tests are run should stay outside of RandomizedTesting code. It is still my intention to get this back into the randomizedtesting project, in which case the test naming patterns should not be assumed here.